es-design-system/pages/molecules/es-pagination.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div>
+        <h1>
+            Pagination
+        </h1>
+        <p>
+            Extended from <b-link href="https://bootstrap-vue.org/docs/components/pagination">
+                bootstrap-vue pagination
+            </b-link>
+        </p>
+        <ul
+            id="paginated-list"
+            class="my-4">
+            <li
+                v-for="item in items.slice(perPage * (currentPage - 1), perPage * currentPage)"
+                :key="item">
+                {{ item }}
+            </li>
+        </ul>
+        <es-pagination
+            v-model="currentPage"
+            :total-rows="rows"
+            :per-page="perPage"
+            list-id="paginated-list" />
```

Comment:
Any guidance from design on horizontal orientation of this component? I think typically it is right aligned on page.

Reply:
Ah right so the prop `align` can control that on the outer level; I added `align="center"` to the docs example to show that option.